pythonph/settings.py

```diff
@@ -43,12 +44,16 @@
 WSGI_APPLICATION = 'pythonph.wsgi.application'
 
 DATABASES = {
+    # 'default': {
+    #     'ENGINE': 'django.db.backends.postgresql_psycopg2',
+    #     'HOST': os.environ.get('POSTGRES_HOST', 'localhost'),
+    #     'NAME': os.environ['POSTGRES_USER'],
+    #     'USER': os.environ['POSTGRES_USER'],
+    #     'PASSWORD': os.environ['POSTGRES_PASSWORD'],
+    # }
```

Comment:
Hi @pandabearcoder you forgot to uncomment the database config.

Reply:
Didn't realized I committed that. Updated!